DTAConfig/Settings/FileSourceDestinationInfo.cs

```diff
@@ -162,6 +186,7 @@ public enum FileOperationOptions
         AlwaysOverwrite,
         OverwriteOnMismatch,
         DontOverwrite,
-        KeepChanges
+        KeepChanges,
+        LinkAsReadOnly
```

Comment:
I have only one comment remains: suggest naming this new option as `AlwaysOverwrite_LinkAsReadOnly`

Reply:
Done by d69c8f8.